plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java

```diff
@@ -33,4 +51,46 @@ public void doGet(Request req, Response rsp, Context ctx) throws IOException {
                                 .toJson(r)
                                 .getBytes(StandardCharsets.UTF_8));
     }
+
+    /** Response for Model server endpoint */
+    public static class SagemakerXgboostResponse {
```

Comment:
Could we rename this class to be made generic. Call it "ExecutionParametersResponse" or some such.